public/app/features/dashboard/displayProfiles/profiles/playlist.ts

```diff
@@ -0,0 +1,18 @@
+import { DashboardDisplayProfile, DisplayProfileMode } from '../types';
+
+export const playlistProfile: DashboardDisplayProfile = {
+  dashNav: {
+    timePicker: DisplayProfileMode.default,
+    title: DisplayProfileMode.default,
+    tvToggle: DisplayProfileMode.default,
+    addPanelToggle: DisplayProfileMode.hidden,
+    dashboardSettingsToggle: DisplayProfileMode.hidden,
+    saveDashboardToggle: DisplayProfileMode.hidden,
+    snapshotToggle: DisplayProfileMode.hidden,
+    starToggle: DisplayProfileMode.hidden,
+    sharePanelToggle: DisplayProfileMode.hidden,
+    customButtons: DisplayProfileMode.hidden,
+  },
+  subMenu: DisplayProfileMode.hidden,
+  sideMenu: DisplayProfileMode.hidden,
+};
```

Comment:
What is the thinking behind this one, you can run a playlist in kiosk mode so wont they conflict?